FeedStoreChallenge/Infrastructure/RealmFeedStore.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  RealmFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Marko Engelman on 09/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+public final class RealmFeedStore: FeedStore {
+	private let realm: Realm
+	
+	public init(configuration: Realm.Configuration) throws {
+		do {
+			realm = try Realm(configuration: configuration)
+		} catch let error {
+			throw error
+		}
```

Comment:
There's no need for the do/catch block here since this method can throw.

```suggestion
		realm = try Realm(configuration: configuration)
```